pip_build.py

```diff
@@ -45,14 +46,37 @@
 ]
 
 
-def build():
+def export_version_string(version, is_nightly=False):
+    """Export Version and Package Name."""
+    if is_nightly:
+        date = datetime.datetime.now()
+        version += f".dev{date.strftime('%Y%m%d%H')}"
+        # Replaces `name="keras"` string in `setup.py` with `keras-nightly`
+        with open("setup.py") as f:
+            setup_contents = f.read()
+        with open("setup.py", "w") as f:
+            setup_contents = setup_contents.replace(
+                'name="keras-nlp"', 'name="keras-nlp-nightly"'
+            )
+            setup_contents = setup_contents.replace(
+                '"tensorflow-text', '"tf-nightly", "tensorflow-text-nightly'
```

Comment:
nit: missing the trailing `"` mark on both sides, which is find just inconsistent with two lines up

Reply:
The reason I did't do that is we have `"tensorflow-text;` at the moment in `setup.py`.  Didn't want to add `;` since it might break when we remove it in `setup.py`

Reply:
make sense!